source/bson-corpus/bson-corpus.rst

```diff
@@ -166,54 +198,90 @@ any tests which are not supported on that platform.
 Testing validity
 ----------------
 
-To test validity of a case in the ``valid`` array, we consider up to four
-possible "input" representations: BSON, "canonical" BSON,
-extended JSON, and "canonical" extended JSON.  (Not all will exist for a
-given case).
+To test validity of a case in the ``valid`` array, we consider up to five
+possible representations:
+
+* Canonical BSON (denoted herein as "cB") -- fully valid, spec-compliant BSON
+
+* Degenerate BSON (denoted herein as "dB") -- invalid but still parseable BSON
+  (bad array keys, regex options out of order)
+
+* Canonical Extended JSON (denoted herein as "cEJ") -- A string format based on
+  the JSON standard that emphasizes type preservation at the expense of
+  readability and interoperability.
+
+* Degenerate Extended JSON (denoted herin as "dEJ") -- An invalid form of
+  Canonical Extended JSON that is still parseable.  (For example, "1e100"
+  instead of "1E+100".)
```

Comment:
A `degenerate_extjson` definition is missing from the "Validity test case keys" section above.